chapters/data-analysis.tex

```diff
@@ -1,182 +1,237 @@
-The cleaning process involves (1) making the data set easily usable and understandable, 
+Loading encrypted data frequently can be disruptive to the workflow.
+To facilitate the handling of the data, remove any personally identifiable information from the data set.
+This will create a de-identified data set, that can be saved in a non-encrypted folder.
+De-identification,\sidenote{\url{https://dimewiki.worldbank.org/wiki/De-identification}}
+at this stage, means stripping the data set of direct identifiers.\sidenote{\url{ https://www.povertyactionlab.org/sites/default/files/resources/J-PAL-guide-to-deidentifying-data.pdf}}
+To be able to do so, you will need to go through your data set and
+find all the variables that contain identifying information.
+Flagging all potentially identifying variables in the questionnaire design stage
+simplifies the initial de-identification process.
+If you haven't done that, that are a few tools that can help you with it.
+JPAL's \texttt{PII scan}, as indicated by its name,
+scans variable names and labels for common string patterns associated with identifying information.\sidenote{
+	\url{https://github.com/J-PAL/PII-Scan}}
+The World Bank's \texttt{sdcMicro}
+lists variables that uniquely identify observations,
+as well as allowing for more sophisticated disclosure risk calculations.\sidenote{
+	\url{http://sdctools.github.io/sdcMicro/articles/sdcMicro.html}}
+The \texttt{iefieldkit} command \texttt{iecodebook}
+lists all variables in a data set and exports an Excel sheet
+where you can easily select which variables to keep or drop.\sidenote{
+	\url{https://dimewiki.worldbank.org/wiki/Iecodebook}}
+
+Once you have a list of variables that contain PII,
+assess them against the analysis plan and ask:
+will this variable be needed for analysis?
+If not, the variable should be dropped.
+Don't be afraid to drop too many variables the first time,
+as you can always go back and remove variables from the list of variables to be dropped,
+but you can not go back in time and drop a PII variable that was leaked
+because it was incorrectly kept.
+Examples include respondent names, enumerator names, interview date, respondent phone number.
+If the variable is needed for analysis, ask:
+can I encode or otherwise construct a variable to use for the analysis that masks the PII,
+and drop the original variable?
+This is typically the case for most identifying information.
+Examples include geocoordinates
+(after constructing measures of distance or area,
+drop the specific location),
+and names for social network analysis (can be encoded to secret and unique IDs).
+If PII variables are strictly required for the analysis itself,
+it will be necessary to keep at least a subset of the data encrypted through the data analysis process.
+If the answer is yes to either of these questions,
```

Comment:
It is not clear what "these questions" mean. All of the above, the ones immediately above. I think this could be done better, but not sure what to suggest as I am not 100% sure what we mean.

Reply:
There are only two questions in the whole chapter, and they are in the same paragraph as this "these questions".  I guess we can say "to either of the two previous questions"?

Reply:
Ok, I think one reason for my confusion was that one sentence separated the line with `these questions` and the actual questions. I moved that sentence to the end of the paragraph, italicized the actual questions, and made some other small changes here: f7e572ac3894a89eec2eef340868a33919c422e0

If you are fine with this, please just resolve this issue.

Reply:
that's great. Thanks, Kris!